### ISSUE TITLE
SMP Imagenet과 Noisynet 비교

### ISSUE THREAD

**Issue by JaegyeongByun** [bcaitech1/p3-ims-obd-simple#4]
model : Unet (encoder : timm-efficientnet-b3)
loss : CrossEntropyLoss
optimizer : Adam
learning rate : 0.0001
weight decay : 1.00E-06
batch size : 16
epoch : 40
random seed : 21
augmentation : noramlize

encoder weight | best epoch | train_loss | val_loss | mIoU (val) | mIoU (LB)
-- | -- | -- | -- | -- | -- 
Imagenet | 37 | 0.05110 | 0.3593 | 0.4803 | 0.4950
Noisynet | 32 | 0.05539 | 0.2969 | 0.5159 | 0.4951

![image](https://user-images.githubusercontent.com/71882533/117030250-5de61400-ad3a-11eb-9683-7d81c2bde615.png)
